galley/pkg/runtime/monitoring/monitoring.go

```diff
@@ -134,6 +191,35 @@ func newView(measure stats.Measure, keys []tag.Key, aggregation *view.Aggregatio
 	}
 }
 
+func getStateTypeConfigKeys() ([]tag.Key, error) {
```

Comment:
does this need to be done over and over?  can we not just reuse the same `[]tag.Key` for each registration?

Reply:
thanks..fixed